staging/src/k8s.io/metrics/pkg/client/clientset_test/clientset_test.go

```diff
@@ -39,3 +43,148 @@ func TestFakeList(t *testing.T) {
 		t.Errorf("Unexpected error: %v", err)
 	}
 }
+
+func TestV1alpha1PodMetricsesList(t *testing.T) {
+	expected := v1alpha1.PodMetricsList{
+		Items: []v1alpha1.PodMetrics{
+			{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "pod-1",
+				},
+				Containers: []v1alpha1.ContainerMetrics{
+					{
+						Usage: corev1.ResourceList{
+							corev1.ResourceCPU:    resource.MustParse("1"),
+							corev1.ResourceMemory: resource.MustParse("1G"),
+						},
+					},
+					{
+						Usage: corev1.ResourceList{
+							corev1.ResourceCPU:    resource.MustParse("2"),
+							corev1.ResourceMemory: resource.MustParse("2G"),
+						},
+					},
+				},
+			},
+		},
+	}
+
+	client := fake.NewSimpleClientset(&expected)
+
+	actual, err := client.MetricsV1alpha1().PodMetricses("").List(metav1.ListOptions{})
+	if err != nil {
+		t.Errorf("Unexpected error: %v", err)
+	}
+
+	if expected.String() != actual.String() {
```

Comment:
`reflect.DeepEquals(expected, actual)` always returns false. Is there a better way to check meaningful equality other than a string comparison?